src/main/java/org/mifos/processor/bulk/properties/TenantImplementation.java

```diff
@@ -0,0 +1,26 @@
+package org.mifos.processor.bulk.properties;
+
+import java.util.HashMap;
+
+public class TenantImplementation {
```

Comment:
I think these are already available in the code base already.

Reply:
This is available in channel but i don't think it is present in bulk